### DISCUSSION TITLE
Can't add a new Matter device to HA

### DISCUSSION THREAD

**Discussion by mforbiz** [openthread/openthread#10868] [General]
Hi.
I’ve succesfully installed Open Tread Border Router on certified Espressif ESP Thread Border Router Board and all works fine:

![image](https://github.com/user-attachments/assets/c9b642ef-9e62-4a53-9457-4f398838c2f9)

Thans I’ve installed Thread + Matter + Open Thread Border Router integrations on my HA last version:

![image](https://github.com/user-attachments/assets/d27aad40-1d0f-4d7c-9eef-f59bb33dcbfe)
On Thread configuration I can see the preferred Thread network:

![image](https://github.com/user-attachments/assets/8c1bacad-cd38-46e2-9870-8ffeef5c21e5)
Than I’m trying to add an ESP Zero code matter device via smartphone and QR code using the Add new matter device feature on HA:

![image](https://github.com/user-attachments/assets/b7019d8f-d3af-4b50-841d-a9d89c0338a2)
But even if all seems works fine at the end I rceive this message and I can’t add the matter device:

![image](https://github.com/user-attachments/assets/6db235ab-7ff2-48d2-8136-36bb3b670ffa)

Where I’m wrong?
Any idea?
Thanks in advance.

**Comment by agners** [comment]
Since this is about Home Assistant and it's Matter/Thread integration best is to post in the Home Assistant community at https://community.home-assistant.io/c/configuration/matter-thread/59.
